### PR TITLE
observatory-stage-move

### PR DESCRIPTION
**Pull request by argl**
## Refractr PR Checklist

JIRA ticket: [link to relevant JIRA or other system ticket]

When creating a PR for Refractr, confirm you've done the following steps for smooth CI and CD experiences:
- [x] Have you updated the relevant YAML in the PR?
- [x] Have you checked the relevant YAML for any possible dupes regarding your domain?
- [x] Have you checked if there are any TLS cert concerns - e.g. if the domain being redirected already exists, and it is being changed to point at Refractr, is a temporary TLS 'outage' while waiting for Lets Encrypt certification via HTTP challenge okay? If not, [have you followed these steps for using DNS challenges with our cert-manager setup](https://mana.mozilla.org/wiki/display/SRE/Refractr+-+How+To+-+DNS+Challenges)?
- [ ] If desired, have you generated the Nginx manually to confirm addition works as expected? 
- [ ] If desired, are you able to connect to EKS (cluster itse-apps-prod-1, namespace fluxcd) to more closely monitor the deploys?

After PR merge, next steps include:
- [ ] If going straight from main merge & Stage deploy to a release & production deploy, create the relevant GitHub release with an incremented version / tag applied.
- [ ] Confirm you are ready and able to perform the requested DNS creation or change post-deploy?

### CODE REVIEW

File: prod-refractr.yml [hunk 1]
```diff
@@ -1285,3 +1285,16 @@ refracts:
...
+
+# MP-1553 HTTP Observatory move for MDN staging environment
+- dsts:
+  - ^/analyze/(.*): 'developer.allizom.org/en-US/observatory/analyze?host=$1'
+  - /faq/: developer.allizom.org/en-US/observatory/docs/faq
+  - /statistics/: developer.allizom.org/en-US/observatory
+  - /terms/: developer.allizom.org/en-US/observatory
+  - /: developer.allizom.org/en-US/observatory
+  srcs: observatory.allizom.org
+  status: 302
```
Comment (caugner) [inline prod-refractr.yml:1297]: Note that we'll want HTTP 301 to get the SEO authority.

**Review by caugner — CHANGES_REQUESTED** [review]
LGTM, but let's use HTTP 301.

PS: You might also want to link the Jira bug in the PR description directly.

**Comment by caugner** [comment]
`chore: redirect observatory.allizom.org to developer.allizom.org/en-US/observatory`

File: prod-refractr.yml [hunk 2]
```diff
@@ -1285,3 +1285,16 @@ refracts:
...
+  tests:
+    - https://observatory.allizom.org/analyze/mdn.dev: https://developer.allizom.org/en-US/observatory/analyze?host=mdn.dev
+    - http://observatory.allizom.org/: https://developer.allizom.org/en-US/observatory/
```
[the remark this answers is not in the document]
Comment (caugner) [reply prod-refractr.yml:1300]: Oh, interesting, do the tests not run on PRs?

[the remark this answers is not in the document]
Comment (argl) [reply prod-refractr.yml]: Fixed. Please review again

**Review by ahoneiser — APPROVED** [review]
lgtm

**Comment by ahoneiser** [comment]
We've just created a new release for refractr ([v0.0.191](https://github.com/mozilla-it/refractr/releases/tag/v0.0.191)), PR needs a rebase :slightly_smiling_face:

**Comment by argl** [comment]
Should be up to date now.

**Pull request merged**